change/@internal-calling-component-bindings-40d08897-fec7-4df5-a3e1-4a3b2e38a505.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Participant button to work during local hold",
```

Comment:
flyby nitty nit: changelog comment should be a short complete sentence (so it can be used directly in the release changelog): `bugfix: participant button should work during local hold` or some such.